stdlib/experimental/influxdb/influxdb_test.go

```diff
@@ -0,0 +1,209 @@
+package influxdb
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func Test_api(t *testing.T) {
+	for _, test := range []struct {
+		name string
+
+		// passed to api func
+		args values.Object
+
+		// expected to be returned from api func call
+		expectedErrorMessage string
+		expectedStatusCode   int64
+		expectedResponseBody []byte
```

Comment:
Can you add the expected headers to this test case? We want test coverage over that part of the API as well.